startup/gui/ocio.py

```diff
@@ -100,6 +100,7 @@ def __plugSet( plug ) :
 def __displayTransformCreator( name ) :
 
 	result = GafferImage.DisplayTransform()
+	result["channels"].setValue( "*[RGB]" )
```

Comment:
I don't quite remember how all this works, but this one needs a different default to the one below?